app/controllers/api/v0/service_plans_controller.rb

```diff
@@ -1,3 +1,5 @@
+require "manageiq-messaging"
```

Comment:
Should this require move down into the `messaging_client` method where it's used?

Reply:
I think that's fine, the specs fail after moving but requiring it in the specs makes them pass again.